lib/cryptoexchange/client.rb

```diff
@@ -8,6 +8,8 @@ def initialize(ticker_ttl: 3)
     def pairs(exchange)
       pairs_classname = "Cryptoexchange::Exchanges::#{StringHelper.camelize(exchange)}::Services::Pairs"
       Object.const_get(pairs_classname).new.fetch
+    rescue HTTP::ConnectionError, HTTP::TimeoutError, JSON::ParserError
+      return { error: 'exchange service temporary unavailable' }
```

Comment:
@Swingcloud How about `#{exchange}'s service is temporarily unavailable.`

Reply:
will do